### PR TITLE
feat: Add optional autocomplete for form header table

### PR DESCRIPTION
**Pull request by dev-marek**
**Issue**

https://github.com/gravitee-io/issues/issues/5157

**Description**

Made key autocomplete optional for table to be reused as other type than HTTP headers.

**Additional context**

---
#### 🧪&nbsp;&nbsp;Gravitee.io Automatic Prerelease @gravitee/ui-schematics
```
npm install @gravitee/ui-schematics@12.19.0-apim-5157-optional-autocomplete-for-form-header-61e5e7f
```
```
yarn add @gravitee/ui-schematics@12.19.0-apim-5157-optional-autocomplete-for-form-header-61e5e7f
```

---
#### 🧪&nbsp;&nbsp;Gravitee.io Automatic Prerelease @gravitee/ui-policy-studio-angular
```
npm install @gravitee/ui-policy-studio-angular@12.19.0-apim-5157-optional-autocomplete-for-form-header-61e5e7f
```
```
yarn add @gravitee/ui-policy-studio-angular@12.19.0-apim-5157-optional-autocomplete-for-form-header-61e5e7f
```

---
#### 🧪&nbsp;&nbsp;Gravitee.io Automatic Prerelease @gravitee/ui-particles-angular
```
npm install @gravitee/ui-particles-angular@12.19.0-apim-5157-optional-autocomplete-for-form-header-61e5e7f
```
```
yarn add @gravitee/ui-particles-angular@12.19.0-apim-5157-optional-autocomplete-for-form-header-61e5e7f
```

---

### CODE REVIEW

File: projects/ui-particles-angular/index.scss [hunk 1]
```diff
@@ -13,4 +13,4 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@forward './src/scss/index.scss';
+@forward './src/scss/index';
```
Comment (ThibaudAV) [inline projects/ui-particles-angular/index.scss:16]: why did you change this ?

Comment (dev-marek) [reply projects/ui-particles-angular/index.scss:16]: I think one of lints made me to do that. I can revert

**Review by ThibaudAV — APPROVED** [review]
